util/reggen/data.py

```diff
@@ -230,10 +232,12 @@ def __init__(self):
         self.addr_width = 12
         self.base_addr = 0
         self.name = ""
+        self.instance = ""
```

Comment:
This should be `inst_name`. Perhaps clarify in a comment what `name` and `inst_name` are.